src/rabbit_auth_backend_http.erl

```diff
@@ -54,9 +54,10 @@ user_login_authorization(Username) ->
         Else                          -> Else
     end.
 
-check_vhost_access(#auth_user{username = Username}, VHost, _Sock) ->
+check_vhost_access(#auth_user{username = Username}, VHost, Sock) ->
     bool_req(vhost_path, [{username, Username},
-                          {vhost,    VHost}]).
+                          {vhost,    VHost},
+			  {ip, inet_parse:ntoa(extract_address(Sock))}]).
```

Comment:
`extract_address/1` can return `undefined`, which will result an error when trying to build the query string.